exonum/Cargo.toml

```diff
@@ -75,3 +76,6 @@ float_serialize = []
 long_benchmarks = []
 metrics-log = []
 sodiumoxide-crypto = ["exonum_sodiumoxide"]
+
+[build-dependencies]
+protoc-rust = "=2.1.4"
```

Comment:
Is its output compatible with `grpc-rs` (aka `grpcio`) that we're going to use for gRPC interactions? Shouldn't we use `protoc-grpcio` instead for Protobuf?

Reply:
@ilammy I guess if we will have a problem with that we will be able to switch to `protoc-grpcio` instead.